good_first_issues/graphql/commands.py

```diff
@@ -40,19 +45,24 @@
     help="View all the issues found without limits.",
     is_flag=True,
 )
-@click.argument("name")
-def gql(name: str, repo: str, user: bool, web: bool, limit: int, all: bool):
+@click.argument("name",required=False)
+def gql(name: str, repo: str, user: bool, web: bool, limit: int, all: bool, hacktoberfest: bool):
     """
     Sub-command for GraphQL mode.
     """
+
+    if name is None and hacktoberfest is False:
+        click.echo("Missing Argument NAME")
+        raise click.Abort()
+
```

Comment:
Arguments are required by default and `click` will show an error message if it is missing. So we don't need to handle it here.  We can also remove the `required` flag from argument.

Image for reference: 
![image](https://user-images.githubusercontent.com/13623913/95541263-a6db2180-0a12-11eb-9fbd-e7967007fb65.png)

Reply:
Hi @yankeexe I added it because, 

'gfi gql --hacktoberfest'

This command does not need argument. This will list all repositories with topic 'hacktoberfest'

Reply:
oh ok , got it. That's pretty neat 🙂 
Can you format the error message like you did on 1b8994c27db3df3fc7e36b58fc3162c3f830ef6a